adding-a-new-module/README.md

```diff
@@ -2,22 +2,23 @@
 This **project** includes all common dependencies and settings for kaluga module.
 
 # Steps for adding a new module
-1. Run gradle task: `./gradlew createNewModule -P module_name=<your-module>` from the root directory.
-1. Include your module to kaluga project edit `kaluga/settings.gradle.kts` by adding: `include(":<your-module>")`.
-1. Add Unit tests
+1. Run gradle task: `./gradlew createNewModule -P module_name=<your-module> -P package_name=<your-package>` from the root directory.
+   * `package_name` can be omitted if the module and package names are the same.
+2. Include your module to kaluga project edit `kaluga/settings.gradle.kts` by adding: `include(":<your-module>")`.
+3. Add Unit tests
     * [Common Unit Tests](#commonTests)
     * [iOS Unit Tests](#iosTests)
     * [Android Unit Tests](#androidLibTests)
-1. [Add Android Instrumented Unit Tests](#instrumentedUnitTests)
```

Comment:
That's `1` ones was to a reason no to track numbers but let markdown render do the work. I mean all 1 will produces 1, 2, 3 for you. And now if you add new item in the middle you have to update all next items manually...

Reply:
yeah in general the `1,1,1` is good IMHO, it discourages someone from writing "see step 2" when that step might be rendered as step 3 or something at some point.

Reply:
Hmm,AS must have automatically changed this, cause I didnt do it. Ill revert